docs/source/devguides/supporting/examples/MatLabAgent.rst

```diff
@@ -0,0 +1,233 @@
+
+5. Run the below command and make a note of the server key. This required in configuring the stand alone agent on windows.
+
+``volttron-ctl auth serverkey``
+
+Configuration Modifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The MatLab agent uses the configuration store to dynamically change inputs.
+More information on the config store and how it used can be found here.
+
+ * :ref:`VOLTTRON Configuration Store <VOLTTRON-Configuration-Store>`
+
+ * :ref:`Agent Configuration Store <ConfigurationStore>`
+
+ * :ref:`Agent Configuration Store Interface <Agent-Configuration-Store-Interface>`
+
+
+
+Setup on Windows
+----------------
+
+Install pre-requisites
+~~~~~~~~~~~~~~~~~~~~~~~
+1. Install python 2.7 from `here <https://www.python.org/downloads/windows/>`__.
+
+2. Install MatLab engine from  `here <https://www.mathworks.com/help/matlab/matlab_external/install-the-matlab-engine-for-python.html>`_.
+
+Install StandAloneMatLab Agent
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The standalone MatLab agent is designed to be usable in a 
+Windows environment. 
+
+.. warning:: 
+
+        VOLTTRON is not designed to run in a Windows environment. 
+        Outside of cases where it is stated to be usable in a 
+        Windows environment, it should be assumed that it will
+        NOT function as expected.
+
+
+1. Download VOLTTRON
+
+   Download the VOLTTRON develop repository from github. Download the zip  
+   from `GitHub <https://github.com/VOLTTRON/volttron/tree/develop>`_ and unzip 
+   
+   |github-image|
+
+2. Setup the PYTHONPATH
+   
+   Open the Windows explorer, and navigate to "Edit environment variables for your account."
+   
+   |cmd-image|
+
+   Select "New"
+   
+   |env-vars-image_1|
+   
+   For "Variable name" enter: "PYTOHNPATH"
+   For "Variable value" either browse to your volttron installation, or enter in the path to your volttron installation.
+   
+   |env-vars-image_2|
+   
+   Select "OK" twice.
+
+3. Set up the environment.
+   
+   Open up the command prompt.
+   
+   |cmd-image_2|
+   
+   Naviage to your volttron installation.
+   
+   ``cd \Your\directory\path\to\volttron-develop``
+   
+   Use pip to install and setup dependencies.
+   
+   ``pip install -r requirements.txt``
+   
+   ``pip install -e .``
+   
+   .. note::
+   
+     If you get the error doing the second step because of an already installed volttron from a different directory, manually delete the volttron-egg.link file from your <python path>\Lib\site-pacakages directory ( for example, del  C:\Python27\lib\site-packages\volttron-egg.link ) and re run the second command
+
+4. Configure the agent
+
+The configurations for the standalone agent is in setting.py in the same directory as the standalone agent.
+
+1. settings.py
+   * 'vip_address' needs to be set to the address of your volttron instance
+   
+   * 'port' needs to be set to the port of your volttron instance
+   
+   * 'server_key' needs to be set to the public server key of your remote platform.
```

Comment:
This command is run on the linux side, so to avoid confusion, link  to last step on Windows setup and say set value of server key to the value got in last step of Linux step.